resources/test/nested-testharness.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+/**
+ * Execute testharness.js and one or more scripts in an iframe. Report the
+ * results of the execution.
+ *
+ * @param {...function|string} bodies - a function body. If specified as a
```

Comment:
Looks from the implementation like it takes any number of strings, and if function objects work it's only if they serialize to a string that will work, but it's not invoking the original function object.

Reply:
That's all accurate. I attempted to communicate the multiplicity via JSDoc's `...` syntax and the function serialization via the parameter's description. Can you suggest how I could make these details more clear?

Reply:
I think just `...string` and a note that serializing a function might do it.